src/select-panel/default-item.tsx

```diff
@@ -15,16 +15,20 @@ const DefaultItemRenderer = ({
   onClick,
   disabled,
 }: IDefaultItemRendererProps) => (
-  <div className={`item-renderer ${disabled ? "disabled" : ""}`}>
+  <label
+    className={`item-renderer ${disabled ? "disabled" : ""}`}
+    htmlFor={option.value}
+  >
     <input
+      id={option.value}
       type="checkbox"
       onChange={onClick}
       checked={checked}
       tabIndex={-1}
       disabled={disabled}
     />
     <span>{option.label}</span>
-  </div>
+  </label>
 );
```

Comment:
use a more semantic HTML element for the label, such as a label element. This would improve the accessibility of the component and make it easier to style